cmd/cue/cmd/testdata/script/get_go_outfile.txtar

```diff
@@ -0,0 +1,44 @@
+# Issue 4108
+
+exec cue get go --outfile outfile.cue ./x
+
+# We can only compare the generated file to the expected golden file,
+# as all imports are skipped and only the given package is generated.
+cmp outfile.cue x_go_gen.cue.golden
+
+-- go.mod --
+module mod.test
+
+go 1.21
+-- cue.mod/module.cue --
+module: "mod.test"
+language: version: "v0.9.0"
+-- x/x.go --
```

Comment:
best to include multiple files in `package x`, to make sure that the joining of declarations is tested.